Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'rails', '~> 5.2'
 gem 'turbolinks'
 
 gem 'recaptcha', require: 'recaptcha/rails'
-
+gem 'rest-client'
```

Comment:
Already specified in [the gemspec](https://github.com/calagator/calagator/blob/d6d672c9fb4c90b63682233dfed6985e55cfcf2d/calagator.gemspec#L47).